### PR TITLE
[FIX] Wrong user in user info

### PR DESCRIPTION
**Pull request by gabriellsh**
## Proposed changes (including videos or screenshots)

Fixed some race conditions in admin.

Self DMs used to be created with the userId duplicated. Sometimes rooms can have 2 equal uids, but it's a self DM. Fixed a getter so this isn't a problem anymore.

### CODE REVIEW

File: client/views/room/MemberListRouter.js [hunk 1]
```diff
@@ -6,6 +6,18 @@ import { useRoom } from './providers/RoomProvider';
...
 import RoomMembers from './contextualBar/RoomMembers';
 
+const getUid = (room, ownUserId) => {
+	if (room.uids.length === 1) {
+		return room.uids.length;
+	}
+
+	const uid = room.uids.filter((uid) => uid !== ownUserId).shift();
+
+	// Self DMs used to be created with the userId duplicated.
```
Comment (dougfabris) [inline client/views/room/MemberListRouter.js:16]: Remove the comments on the code and use this nice information on the PR's description

**Review by dougfabris — CHANGES_REQUESTED** [review]
Remove the comments on the code and use this nice information on the PR's description.

**Comment by gabriellsh** [comment]
> Remove the comments on the code and use this nice information on the PR's description.

IMHO, this information is relevant enough to be in the code. I fear someone might want to change this without the proper consideration. WDYT?

I can add it to the PR also though, no problem.

**Comment by dougfabris** [comment]
> > Remove the comments on the code and use this nice information on the PR's description.
> 
> IMHO, this information is relevant enough to be in the code. I fear someone might want to change this without the proper consideration. WDYT?
> 
> I can add it to the PR also though, no problem.

If you think it's a piece of relevant information to be there, go ahead!

**Review by dougfabris — DISMISSED** [review]

**Comment by gabriellsh** [comment]
I agree! We shouldn't need to explain code, I'm all for that. But I think in specific cases it's needed, since we can't fix the "root" problem and this is a workaround. The comments don't explain the code itself, as it's pretty clear what it does. It explains why that was the choice.

I'm not trying to justify the comments BTW, thats just my opinion. Maybe @ggazzo can comment about this too, since both our POVs are relevant in this case.

**Review by ggazzo — APPROVED** [review]

**Pull request merged**